### ISSUE TITLE
Repeating "Service unavailable" and "Operation cancelled" error messages sent all client concerning specific files. Keeps reproducing with new files.

### ISSUE THREAD

**Issue by geoffm** [owncloud/core#25481]
### Actual behaviour
OC Client shows
> Error downloading X - server replied: Service Unavailable
> Operation canceled
> Operation canceled
> Operation canceled
> File X - server replied: Service Unavailable
> File Y: Operation Cancelled
> File Z: Operation Cancelled

The messages appear for all clients sharing the files.

### Server configuration
**Operating system**:
Debian 7
**Web server:**
Apache
**Database:**
MySQL
**PHP version:**
5.6
**ownCloud version:** (see ownCloud admin page)
9.0.3
**Updated from an older ownCloud or fresh install:**
updated from 9.0.1. This bug was also present in 9.0.1.

**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**
```
 - activity: 2.2.1
  - dav: 0.1.6
  - federatedfilesharing: 0.1.0
  - files: 1.4.4
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
```

**The content of config/config.php:**
```
  'instanceid' => 'ochs1tn1mpbb',
  'passwordsalt' => 'Ij1HYokETCkuou8EzDivgRLJGsNu2B',
  'secret' => 'PDd9k+9FuA/XB4XxWZCUqKzn+hRYtnR1MDKdorKcX5D5we0c',
  'trusted_domains' =>
  array (
    0 => 'nuage.cetab.org',
  ),
  'datadirectory' => '/var/nuage',
  'overwrite.cli.url' => 'https://nuage.cetab.org',
  'dbtype' => 'mysql',
  'version' => '9.0.3.2',
  'dbname' => 'oc_nuage',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'oc_user',
  'dbpassword' => 'XupSKhjDcuTqYBr7',
  'logtimezone' => 'America/Toronto',
  'installed' => true,
  'default_language' => 'fr',
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'mail_smtpmode' => 'smtp',
  'loglevel' => 4,
  'mail_smtpauthtype' => 'LOGIN',
  'mail_from_address' => 'nuage',
  'mail_domain' => 'cetab.org',
  'mail_smtphost' => 'smtp.elasticemail.com',
  'mail_smtpport' => '2525',
  'appstore.experimental.enabled' => true,
  'singleuser' => false,
  'maintenance' => false,
  'mail_smtpsecure' => 'tls',
  'mail_smtpauth' => 1,
  'mail_smtpname' => 'info@cetab.org',
  'mail_smtppassword' => 'a755a429-82eb-480a-a0e9-3958d466a65f',
  'updatechecker' => false,
  'theme' => '',
  'htaccess.RewriteBase' => '/',
);
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**

**Operating system:**
Windows
#### ownCloud log (data/owncloud.log)
```
{"reqId":"3wXBVmwsw9+RYWOHILNW","remoteAddr":"192.222.172.176","app":"PHP","mes$
{"reqId":"yRqhsNtBoQxlt+poP3Fz","remoteAddr":"","app":"cron","message":"Not ins$
{"reqId":"Gl5FmU2ltWIXJgQad5QT","remoteAddr":"66.131.21.53","app":"mysql.setup"$
{"reqId":"Gl5FmU2ltWIXJgQad5QT","remoteAddr":"66.131.21.53","app":"mysql.setup"$
{"reqId":"Gl5FmU2ltWIXJgQad5QT","remoteAddr":"66.131.21.53","app":"PHP","messag$
```
![2016-07-14 15_17_35-owncloud](https://cloud.githubusercontent.com/assets/901518/16852449/2fe54c96-49d6-11e6-85a0-2ee40eed7629.png)

**Comment by PVince81** [comment]
Thanks for the detailed report. I don't see anything wrong in your configuration.

Does the problem only happen with files that have special characters in the path ?
Please set "loglevel" to 0 in your config.php then check the log again when the error occurs, and post it here if there is anything.

**Comment by geoffm** [comment]
Thanks for coming back to me @PVince81 .
I don't think it only happens only with special characters, and for sure it doesn't happen with all files with special characters. I have been paying attention trying to find a common denominator to the affected files and folders, and user behaviors regarding their creation and upload, and found nothing yet.
However, I think I have solved the problem by running occ --file:scan from the server terminal. All error messages disappeared from all users after that. They hadn't come back so far (after about a week) and for sure it's the first thing I will try again if they do.

**Comment by PVince81** [comment]
@geoffm hmm, then if the scan seems to fix it it means there was a discrepancy between the index (oc_filecache) and what was actually on disk. But still, that shouldn't result in "service unavailable".

**Comment by geoffm** [comment]
ok. well, will try to share a detailed log should this phenomena manifest again.

**Comment by PVince81** [comment]
If you don't mind I'll close this then. Feel free to reopen if it happens again, thanks.

**Comment by MONMMK** [comment]
Hi @PVince81 

My loglevel is set to "0". But the error is still there. Please help me.

Operating system: Ubuntu Linux 14.04.3
Web server: Apache
Database: MySql
PHP Version: 4.0.10
Owncloud Version: 8.1.9

**Comment by lubbie** [comment]
This is definitely a bug in the system. My OwnCloud has been down for weeks due to this issue and this post saved me. After I ran the files:scan commend the issue resolved itself. 

I am running OwnCloud 8.2.2

**Comment by eehmke** [comment]
I run owncloud 10.0 on a debian 8 system. I had to delete an old user because he had forgotten his password. He had no private data. I recreated the user and added him to my group. He can now log in via the web gui and see all files, but when he tries to download some files (pdf) he gets "Service unavailable". In the log is: 
{"reqId":"9ukgKVpXE6YQ3k1u50fu","level":4,"time":"2017-05-03T11:34:07+02:00","remoteAddr":"2003:88:680f:39fc:1058:d19d:5016:b80","user":"bernd","app":"webdav","method":"GET","url":"\/remote.php\/webdav\/Ausbildung\/Erweiterungspr%C3%BCfung%20TMG%20Fragen%20und%20L%C3%B6sungen%20m%C3%BCndl%20Pr%C3%BCfung.pdf?downloadStartSecret=i1v2trel53","message":"Exception: {\"Message\":\"HTTP\\\/1.1 503 Encryption not ready: multikeydecrypt with share key failed:error:0407109F:rsa routines:RSA_padding_check_PKCS1_type_2:pkcs decoding error\",\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\ServiceUnavailable\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/composer\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/CorePlugin.php(85): OCA\\\\DAV\\\\Connector\\\\Sabre\\\\File->get()\\n#1 [internal function]: Sabre\\\\DAV\\\\CorePlugin->httpGet(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#2 \\\/var\\\/www\\\/owncloud\\\/lib\\\/composer\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/lib\\\/composer\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(479): Sabre\\\\Event\\\\EventEmitter->emit('method:GET', Array)\\n#4 \\\/var\\\/www\\\/owncloud\\\/lib\\\/composer\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(254): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#5 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v1\\\/webdav.php(63): Sabre\\\\DAV\\\\Server->exec()\\n#6 \\\/var\\\/www\\\/owncloud\\\/remote.php(165): require_once('\\\/var\\\/www\\\/ownclo...')\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/lib\\\/Connector\\\/Sabre\\\/File.php\",\"Line\":324,\"User\":\"bernd\"}"}

**Comment by kdslkdsaldsal** [comment]
> And it should be documented so that people like me will find it before starting to write issues.

If there is something to document here you could post a new issue in the documentation tracker with the infos you would like to see:

https://github.com/owncloud/documentation/issues

**Comment by ownclouders** [comment]
Hey, this issue has been closed because the label `needs info` is set and there were no updates for 14 days. Feel free to reopen this issue if you deem it appropriate.

(This is an automated comment from [GitMate.io](https://gitmate.io).
